tests/test-cases/Group1-Docker-Commands/1-40-Docker-Restart.robot

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation   Test 1-40 - Docker Restart
+Resource        ../../resources/Util.robot
+Suite Setup     Install VIC Appliance To Test Server
+Suite Teardown  Cleanup VIC Appliance On Test Server
+
+*** Keywords ***
+
+Create test containers
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+    Set Environment Variable  RUNNER  ${output}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+    Set Environment Variable  CREATOR  ${output}
+
+*** Test Cases ***
+Restart Running Container
+    Create test containers
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} restart %{RUNNER}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  %{RUNNER}
```

Comment:
should we check the state as a sanity check after clean return? Assert that the expected state is true rather than just rely on `restart` to do the correct thing.